R/summaries.R

```diff
@@ -12,36 +12,54 @@
 #' @param method Specify the method to determine top taxa. Either sum, mean,
 #'   median or prevalence. Default is 'mean'.
 #'
-#' @param abund_values a \code{character} value to select an
+#' @param abund_values A \code{character} value to select an
 #'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}}
 #'
+#' @param ... Additional arguments not used.
+#'
 #' @details
-#' \code{getTopTaxa} extracts the most \code{top} abundant \dQuote{FeatureID}s
+#' The \code{getTopTaxa} extracts the most \code{top} abundant \dQuote{FeatureID}s
 #' in a \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
 #' object.
 #'
 #' @return
-#' For \code{getTopTaxa}: A vector of the most \code{top} abundant
+#' The \code{getTopTaxa} returns a vector of the most \code{top} abundant
 #' \dQuote{FeatureID}s
 #'
 #' @seealso
 #' \code{\link[=getPrevalence]{getPrevalentTaxa}}
 #'
-#' @name getTopTaxa
+#' @name summaries-basic
```

Comment:
So what is an non basic summary? Is there a plan which distinguishes basic and non-basic?

I know, that the questions might come across as rhetorical, but they are not. The aim is to get to the bottom of this and thus keep it simple. It also implies, that if there is the need to distinguish summaries and have different man pages, this is totally fine. But if it is currently not needed, there is no need for two man pages or one man page with a too complicated name.

Reply:
Agreed. I will rename this to `summaries`